frontend/src/components/ContributionMessages/slots/IsNotModerator.spec.js

```diff
@@ -0,0 +1,49 @@
+import { mount } from '@vue/test-utils'
+import IsNotModerator from './IsNotModerator.vue'
+
+const localVue = global.localVue
+
+describe('IsNotModerator', () => {
+  let wrapper
+
+  const mocks = {
+    $t: jest.fn((t) => t),
+    $d: jest.fn((d) => d),
+  }
+
+  const propsData = {
+    message: {
+      id: 113,
+      message: 'asda sdad ad asdasd ',
+      createdAt: '2022-08-29T12:25:34.000Z',
+      updatedAt: null,
+      type: 'DIALOG',
+      userFirstName: 'Bibi',
+      userLastName: 'Bloxberg',
+      userId: 108,
+      __typename: 'ContributionMessage',
+    },
+  }
+
+  const Wrapper = () => {
+    return mount(IsNotModerator, {
+      localVue,
+      mocks,
+      propsData,
+    })
+  }
+
+  describe('mount', () => {
+    beforeEach(() => {
+      wrapper = Wrapper()
+    })
+
+    it('has a DIV .slot-is-not-moderator', () => {
+      expect(wrapper.find('div.slot-is-not-moderator').exists()).toBe(true)
+    })
+
+    it('props.message.default', () => {
```

Comment:
What do you want to test with this? There are more cases

Reply:
in the coverage there was a note at the point where the props object is created as default {}.
 then it wants to have tested it. i tried that. but i only found it somewhere on the net.